### PR TITLE
Add palette usage documentation

### PR DESCRIPTION
**Pull request by amouhoub**
In this PR, we update the documentation to describe the usage of the Palette in the Studio.

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1582,15 +1582,19 @@ NOTE: Please follow the graphiql endpoint: `http://localhost:8080/catalog/graphi
 == Usage of Catalog from GUI
 
 The GUI interaction with the Catalog can be done in two places: Studio portal and Catalog Portal.
-The portals follow the concepts of the Catalog: Workflows store inside buckets, the Workflow can have several revisions, each Workflow have specific description.
-The view of Workflows from Catalog in Studio and from Catalog Portal follow the same view logic. The page is separated in 3 sections.
-The approach is selecting the needed information from left to right. First user should choose the required bucket, then the user can observe all the Workflows in the selected bucket. When the user select the Workflow it will be shown all existing revisions of the Workflow. By selecting a revision you will be able to see the the description about the workflow revision.
+The portals follow the concepts of the Catalog: Workflows are stored inside buckets, a workflow has some metadata and can have several revisions.
+
+The Catalog view in the Studio or in the Catalog Portal allow to browse all the workflows in the Catalog grouped by buckets and to list all the revisions of a workflow along with their commit message.
```
Comment (mboussaa) [inline src/docs/user/ProActiveUserGuide.adoc]: allows to

Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: allow**s** to browse

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1582,15 +1582,19 @@ NOTE: Please follow the graphiql endpoint: `http://localhost:8080/catalog/graphi
 == Usage of Catalog from GUI
 
 The GUI interaction with the Catalog can be done in two places: Studio portal and Catalog Portal.
```
Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: "portal" and "Portal", maybe use capital in both cases?

File: src/docs/user/ProActiveUserGuide.adoc [hunk 3]
```diff
@@ -1582,15 +1582,19 @@ NOTE: Please follow the graphiql endpoint: `http://localhost:8080/catalog/graphi
...
-The view of Workflows from Catalog in Studio and from Catalog Portal follow the same view logic. The page is separated in 3 sections.
-The approach is selecting the needed information from left to right. First user should choose the required bucket, then the user can observe all the Workflows in the selected bucket. When the user select the Workflow it will be shown all existing revisions of the Workflow. By selecting a revision you will be able to see the the description about the workflow revision.
+The portals follow the concepts of the Catalog: Workflows are stored inside buckets, a workflow has some metadata and can have several revisions.
+
+The Catalog view in the Studio or in the Catalog Portal allow to browse all the workflows in the Catalog grouped by buckets and to list all the revisions of a workflow along with their commit message.
 
 === Studio
-Inside the Studio portal there is a `Catalog` menu from which a user can directly interact with the Catalog storage.
+Inside the Studio portal there is a `Catalog` menu from which a user can directly interact with the Catalog to import or publish Workflows.
```
Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: Same here, we should use consistently either "portal" or "Portal".

File: src/docs/user/ProActiveUserGuide.adoc [hunk 4]
```diff
@@ -1582,15 +1582,19 @@ NOTE: Please follow the graphiql endpoint: `http://localhost:8080/catalog/graphi
...
 
 image::studio_catalog_menu.png[align=center]
 
+Besides, the Palette of the Studio lists the user's favourite buckets to allow easy import of workflows from the Catalog with a simple Drag & Drop.
```
Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: I'd use "Additionally" or "In addition"  instead. "Besides" is not commonly at the beginning of a paragraph.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 5]
```diff
@@ -1604,33 +1608,32 @@ When the Workflow is selected from Catalog you have two options:
 
 2. `Append to current Workflow`: Append the selected Workflow from the Catalog to the Workflow already opened inside the Studio.
 
+==== Add Bucket Menu to the Palette
+The Palette makes it easier to use specific templates stored in a specific Bucket of the Catalog when designing a Workflow.
```
Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: I'd remove one of the two "specific", but it is just an opinion.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 6]
```diff
@@ -1604,33 +1608,32 @@ When the Workflow is selected from Catalog you have two options:
...
 
-Add a new menu by clicking on the `Catalog` → `Add Bucket as Extra Catalog Menu` entry. See image below. 
+You can repeat the operation as many times as you wish in order to add other specific Menus you might need for your workflow design. See the image below.
 
-image::studio-catalog-main-menu3.png[align=center]
+image::studio-catalog-main-menu5.png[align=center]
 
-The selected deep-learning-workflows Bucket now appears as a menu in the Studio. See image below.
+==== Change Palette Preset
+The Studio offers the possibility to set a Preset for the Palette. This makes it easier to load the palette with a predefined set of default buckets.
```
Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: Cool add!

File: src/docs/user/ProActiveUserGuide.adoc [hunk 7]
```diff
@@ -1604,33 +1608,32 @@ When the Workflow is selected from Catalog you have two options:
...
+Presets are generally arranged by theme (Basic Examples, Machine Learning, Deep Learning, Big Data, etc.)
+The palette preset can be changed by clicking `View` → `Change Palette Preset`. This will show a window with the actual list of presets as in the image below.
```
Comment (luispinedamx) [inline src/docs/user/ProActiveUserGuide.adoc]: - **P**alette
- with the **current** list of presets

**Pull request merged**